src/main/java/io/stargate/sgv2/jsonapi/service/operation/tasks/BaseTask.java

```diff
@@ -59,6 +59,9 @@ public abstract class BaseTask<
   /** The current result supplier, supplied by the subclass every time we try / re-try */
   private ResultSupplierT resultSupplier = null;
 
+  /** The command context for the current execution, set when execute() is called */
```

Comment:
This is passed as a param to execute(), seems odd to then store it as instance state.

If it is not set in the ctor, then it adds risks that we may use it when we do not know if it has been set. So we either need to check if it is null or rely on the idea that code always runs in a particular way. 

I see we are using this because it is used when building the error handler, I think this is another sign that we need to look at the idea of passing this info into the error handler.